docs/tutorials/contributing/strategy/writing_the_new_strategy.rst

```diff
@@ -7,7 +7,7 @@ Identify a new strategy
 The library has a large number of strategies::
 
     >>> import axelrod as axl
-    >>> len(axl.strategies)
+    >>> len(axl.strategies) # doctest: +SKIP
     103
 
 If you're not sure if you have a strategy that has already been implemented
```

Comment:
Not sure I understand why we should skip this?

Is it not a nice thing to have and up to date? If it's already implemented it'll fall out of the review?